app/actions/websocket/channel.ts

```diff
@@ -328,7 +328,7 @@ export async function handleUserAddedToChannelEvent(serverUrl: string, msg: any)
 
             loadCallForChannel(serverUrl, channelId);
         } else {
-            const addedUser = getUserById(database, userId);
```

Comment:
@larkox I noticed this was checking against the existence of the returned promise instead of the user. Could this potentially be the cause of some issues we've seen with user added to channel events?

Reply:
Nice catch!!

Reply:
Great catch indeed. About what it might solve, I don't think it is any "terrible" issue. It is just the logic to get a missing user that has been added to the channel. And that is probably "autofixed" by the at mention in the "added to channel" post.